### ISSUE TITLE
Problem with compiling for iPad in Xcode

### ISSUE THREAD

**Issue by NN708** [microsoft/MixedReality-SpectatorView#346]
I'm trying to develop Spectator View on iPad for HoloLen 2, following [Spectator View Mobile Setup](https://github.com/microsoft/MixedReality-SpectatorView/blob/master/doc/SpectatorView.Setup.md). I'm using QR code detection for spatial alignment. However, I encountered some errors when I am trying to compile the generated Xcode project. Here's the error message:
```
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_ARObjectScanningConfiguration", referenced from:
      objc-class-ref in ARKitNativeObjectDetection.o
  "_OBJC_CLASS_$_ARObjectAnchor", referenced from:
      objc-class-ref in ARKitNativeObjectDetection.o
  "_OBJC_CLASS_$_ARPointCloud", referenced from:
      objc-class-ref in ARPointCloud.o
  "_OBJC_CLASS_$_AREnvironmentProbeAnchor", referenced from:
      objc-class-ref in ARSessionNative.o
      objc-class-ref in AREnvironmentProbe.o
  "_OBJC_CLASS_$_ARWorldTrackingConfiguration", referenced from:
      objc-class-ref in ARSessionNative.o
  "_OBJC_CLASS_$_ARReferenceImage", referenced from:
      objc-class-ref in ARSessionNative.o
  "_OBJC_CLASS_$_AROrientationTrackingConfiguration", referenced from:
      objc-class-ref in ARSessionNative.o
  "_OBJC_CLASS_$_ARImageAnchor", referenced from:
      objc-class-ref in ARSessionNative.o
  "_OBJC_CLASS_$_ARAnchor", referenced from:
      objc-class-ref in ARSessionNative.o
  "_OBJC_CLASS_$_ARWorldMap", referenced from:
      objc-class-ref in ARWorldMap.o
      objc-class-ref in ARSessionNative.o
      objc-class-ref in ARKitNativeObjectDetection.o
  "_OBJC_CLASS_$_ARPlaneAnchor", referenced from:
      objc-class-ref in ARSessionNative.o
  "_OBJC_CLASS_$_ARSession", referenced from:
      objc-class-ref in ARSessionNative.o
  "_OBJC_CLASS_$_ARReferenceObject", referenced from:
      objc-class-ref in ARWorldMap.o
      objc-class-ref in ARSessionNative.o
      objc-class-ref in AREnvironmentProbe.o
      objc-class-ref in ARKitNativeObjectDetection.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Here's my environment:
OS: macOS Catalina 10.15.3
Unity: 2019.3.2f1
Target: iPad Pro 2018 running iPadOS 13.3.1

Thanks in advance!

**Comment by NN708** [comment]
Solved by using Unity 2018.4.18f1.

**Comment by yinhun2012** [comment]
> Solved by using Unity 2018.4.18f1.

how it solved?i use 2018.4.18f1 lts version,still 100 error defined symbols,i worked for spectator view a hole week,i am going crazy,but still everything error.
